infrastructure/oss/issue.go

```diff
@@ -170,21 +44,22 @@ func toIssue(
 	learnService learn.Service,
 	ep error_reporting.ErrorReporter,
 ) snyk.Issue {
-	title := issue.Title
-
-	if config.CurrentConfig().Format() == config.FormatHtml {
-		title = string(markdown.ToHTML([]byte(title), nil, nil))
-	}
-	remediationAdvice := getRemediationAdvice(issue)
-
 	// find all issues with the same id
-	matchingIssues := []ossIssue{}
+	matchingIssues := []snyk.OssIssueData{}
```

Comment:
This is just re-ordering of the code so that we can use the `additionalData` to compute the remediation advice